config/jobs/cert-manager/cert-manager/cert-manager-presubmits.yaml

```diff
@@ -36,6 +36,38 @@ presubmits:
           - name: ndots
             value: "1"
 
+  - name: pull-cert-manager-make-test
+    always_run: false
+    optional: true
+    context: pull-cert-manager-make-test
+    max_concurrency: 8
+    agent: kubernetes
+    decorate: true
+    branches:
+    - master
+    - release-1.8
+    annotations:
+      testgrid-create-test-group: 'true'
+      testgrid-dashboards: jetstack-cert-manager-presubmits-blocking
+      description: Runs 'bazel test --jobs=1 //...'
```

Comment:
suggestion: this annotation seems to be out of date

Reply:
Fixed in 9cf858c.

Reply:
In the latest version I'm looking at now, the description here still refers to bazel; is there a missing commit?

Reply:
Oops, I missed this `description:`. 06ca5be should be the right one.